segmentation/include/pcl/segmentation/extract_clusters.h

```diff
@@ -251,16 +251,16 @@ namespace pcl
     Indices nn_indices;
     std::vector<float> nn_distances;
     // Process all points in the indices vector
-    for (std::size_t i = 0; i < indices.size (); ++i)
+    for (const auto& point : indices)
```

Comment:
Maybe use a different name, like `point_idx` or `point_index`? `point` sounds like it is an actual point, but it is an index.

Reply:
Definitely yes! I'll fix it.